### PR TITLE
[core] Remove perfect scrollbar

### PR DESCRIPTION
**Pull request by svenefftinge**
#### What it does

Removes perfect-scrollbar.

The reason we introduced it was because of the limited
styling capabilities in Friefox. This has changed end of last year,
as it now supports the properties `scrollbar-color`
and `scrollbar-width`.

#### How to test
Check scrolling behavior and visual experience in chrome and firefox.

#### Review checklist

- [ ] as an author, I have thoroughly tested my changes and carefully followed [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#requesting-a-review)

#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

### CODE REVIEW

**Comment by vince-fugnitto** [comment]
I believe it closes #6509

**Comment by akosyakov** [comment]
(out-of-scope) Would it be possible to apply this styles to webviews as well:
Screen Shot 2019-11-25 at 20 33 54

to close https://github.com/eclipse-theia/theia/issues/5040#issuecomment-551088113 They should go here: https://github.com/eclipse-theia/theia/blob/98ac397e05d8f86e0fe1d4284495ea004428c703/packages/plugin-ext/src/main/browser/webview/pre/main.js#L69

**Comment by kittaakos** [comment]
[bug]: I cannot scroll in the Git History widget in FF.

**Comment by kittaakos** [comment]
[bug]: The scrollbar does not work in the SIW. (FF)
screencast 2019-11-26 08-33-14

**Comment by kittaakos** [comment]
> The scrollbar does not work in the SIW

It is broken in the `Explorer` too. (FF)

**Comment by kittaakos** [comment]
All `scrollContainer`s seem to be broken in FF. See the `Problems` view:
screencast 2019-11-26 08-36-39

**Comment by svenefftinge** [comment]
> Firefox no longer displays the scrollbar for tabs when multiple editors are opened for instance:

This is intentional, as the 'thin' horizontal scrollbar is FF is still too big.

**Comment by vince-fugnitto** [comment]
> > Firefox no longer displays the scrollbar for tabs when multiple editors are opened for instance:
> 
> This is intentional, as the 'thin' horizontal scrollbar is FF is still too big.

If it is in fact intentional I'm not sure we should completely remove it.  The scrollbar is still present in other browsers, and is also present in VS Code.

**Comment by kittaakos** [comment]
> Firefox no longer displays the scrollbar for tabs

It is not just about the missing scrollbar, but the flickering I can see each time I open a new file in the editor. (FF)

Before:
no-flicker

After:
flicker

(Maybe it is not visible in the attached GIF 😕)

File: packages/messages/src/browser/notification-center-component.tsx [hunk 1]
```diff
@@ -67,13 +65,11 @@ export class NotificationCenterComponent extends React.Component<NotificationCen
                         </ul>
                     </div>
                 </div>
-                <PerfectScrollbar className='theia-notification-list-scroll-container'>
```
Comment (AlexTugarev) [inline packages/messages/src/browser/notification-center-component.tsx]: `theia-notification-list-scroll-container` should be recreated without `ps` then.

File: packages/messages/src/browser/notification-center-component.tsx [hunk 2]
```diff
@@ -67,13 +65,11 @@ export class NotificationCenterComponent extends React.Component<NotificationCen
...
-                    <div className='theia-notification-list'>
-                        {this.state.notifications.map(notification =>
-                            <NotificationComponent key={notification.messageId} notification={notification} manager={this.props.manager} />
-                        )}
-                    </div>
-                </PerfectScrollbar>
+                <div className='theia-notification-list'>
```
Comment (AlexTugarev) [inline packages/messages/src/browser/notification-center-component.tsx]: please wrap this in an additional `<div className='theia-notification-list-scroll-container'>` container.

**Pull request closed without merging**